superset/databases/commands/create.py

```diff
@@ -63,7 +67,10 @@ def run(self) -> Model:
             security_manager.add_permission_view_menu("database_access", database.perm)
             db.session.commit()
         except DAOCreateFailedError as ex:
-            logger.exception(ex.exception)
+            event_logger.log_with_context(
+                action=f"db_creation_failed.{ex.__class__.__name__}",
+                engine=database.db_engine_spec.__name__,
```

Comment:
@hughhhh what did you want to do about making the engine string consistent around each of these logs so that we can group by the same engine later when observing the logs?

Reply:
I ended up changing all the engine references in the `event_logger.log_with_context` to pull data from the `database.db_engine_spec.__name__`